cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1879,29 +1879,35 @@ pProtocol =
 
     pEpochSlots :: Parser EpochSlots
     pEpochSlots =
-      EpochSlots <$>
-        ( Opt.option Opt.auto
-            (  Opt.long "epoch-slots"
-            <> Opt.metavar "NATURAL"
-            <> Opt.help "The number of slots per epoch (default is 21600)."
-            )
-        <|>
-          -- Default to the mainnet value.
-          pure 21600
+        ( EpochSlots <$>
+            Opt.option Opt.auto
+              (  Opt.long "epoch-slots"
+              <> Opt.metavar "NATURAL"
+              <> Opt.help "The number of slots per epoch (default is 21600)."
+              )
         )
+      <|>
+        -- Default to the mainnet value.
+        pure defaultEpochSlots
 
     pSecurityParam :: Parser SecurityParam
     pSecurityParam =
-      SecurityParam <$>
-        ( Opt.option Opt.auto
-            (  Opt.long "security-param"
-            <> Opt.metavar "NATURAL"
-            <> Opt.help "The security parameter (default is 2160)."
-            )
-        <|>
-          -- Default to the mainnet value.
-          pure 2160
+        ( SecurityParam <$>
+            Opt.option Opt.auto
+              (  Opt.long "security-param"
+              <> Opt.metavar "NATURAL"
+              <> Opt.help "The security parameter (default is 2160)."
+              )
         )
+      <|>
+          -- Default to the mainnet value.
+          pure defaultSecurityParam
+
+    defaultEpochSlots :: EpochSlots
+    defaultEpochSlots = EpochSlots 21600
```

Comment:
```suggestion
    defaultByronEpochSlots :: EpochSlots
    defaultByronEpochSlots = EpochSlots 21600
```
since it's different for Shelley. And then obviously rename the use sites too.